src/test/java/create_task_use_case/CreateTaskControllerTest.java

```diff
@@ -0,0 +1,26 @@
+package create_task_use_case;
+
+import entities.TaskFactory;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CreateTaskControllerTest {
+    private final static CreateTaskPresenter createTaskPresenter = new CreateTaskPresenter();
+    private final static CreateTaskDataAccess createTaskDataAccess = new CreateTaskDataAccess();
+    private final static TaskFactory taskFactory = new TaskFactory();
+
+    @Test
+    public void testCreateTask(){
```

Comment:
Agreed. This can be done with existsByTitle to check if the task is there, although you'd need to access the days class attribute directly if you wanted to check that the completion status is correct.

Reply:
Hi all, no need to directly access the days class, I will use the createTaskDataAccess to validate the method as desired.

Reply:
My message was supposed to be referring to using the getDays() method from createTaskDataAccess specifically. Apologies if that was unclear. 

PS: class attribute refers to days being Static, thus an attribute of the class weekDataAccess as opposed to an attribute of an instance of weekDataAccess (an instance attribute).